<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdWindow</class>
 <widget class="QDialog" name="ThirdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="itemsListWidgets">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_All_Purchased_items">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>All Purchased Items</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_buy">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy items </string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_recharge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Recharge Balance</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_balance">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Balance">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance : </string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_Total_Price">
      <property name="text">
       <string>Total Price :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="alertLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
